clinica/pipelines/t1_freesurfer_atlas/t1_freesurfer_atlas_utils.py

```diff
@@ -84,14 +130,28 @@ def write_tsv_files(subject_dir: str, image_id: str, atlas: str) -> str:
     from clinica.utils.freesurfer import generate_regional_measures_alt
     from clinica.utils.stream import cprint
 
-    if os.path.isfile(os.path.join(subject_dir, image_id, "mri", "aparc+aseg.mgz")):
-        generate_regional_measures_alt(subject_dir, image_id, atlas)
-    else:
-        cprint(
-            msg=(
-                f"{image_id.replace('_', ' | ')} does not contain "
-                f"mri/aseg+aparc.mgz file. Creation of regional_measures/ folder will be skipped."
-            ),
-            lvl="warning",
-        )
+    if len(image_id.split("_")) == 2:
+        if os.path.isfile(os.path.join(subject_dir, image_id, "mri", "aparc+aseg.mgz")):
+            generate_regional_measures_alt(subject_dir, image_id, atlas)
+        else:
+            cprint(
+                msg=(
+                    f"{image_id.replace('_', ' | ')} does not contain "
+                    f"mri/aseg+aparc.mgz file. Creation of regional_measures/ folder will be skipped."
+                ),
+                lvl="warning",
+            )
+    elif len(image_id.split("_")) == 3:
```

Comment:
Does Omar's comment on testing for `long` apply here too?

Reply:
Yes it does !